clang/test/CodeGenSYCL/addrspace_ctor_dtor.cpp

```diff
@@ -0,0 +1,31 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -disable-llvm-passes %s -emit-llvm -o -  | FileCheck %s
```

Comment:
Emitting llvm IR without passing spir-based triple will result in llvm_unreachable somewhere in CodeGen, AFAIK.
BTW, think we need to rename test, so the name will reflect to changes related to used attribute.

Reply:
Done.